javascript-data-model/order-history.js

```diff
@@ -0,0 +1,68 @@
+var orderHistory = [
+  {
+    number: '114-3941689-8772232',
+    total: '$34.00',
+    numberItems: 1,
+    orderDate: 'August 4, 2020',
+    deliveryDate: 'August 8, 2020',
+    recipient: 'JS Masher',
+    itemDescription: ['book'],
+    itemName: ['Javascript for impatient programmers'],
+    subtotal: ['$31.55'],
+    buyAgain: true,
+    viewItem: true,
+    productReview: true,
+    productSupport: false
+  },
+  {
+    number: '113-9984268-1280257',
+    total: '$44.53',
+    numberItems: 1,
+    orderDate: 'July 19, 2020',
+    deliveryDate: 'July 20, 2020',
+    recipient: 'JS Masher',
+    itemDescription: ['book'],
+    itemName: ['The Timeless Way of Building'],
+    subtotal: ['$41.33'],
+    buyAgain: true,
+    viewItem: true,
+    productReview: true,
+    productSupport: false
+  },
+  {
+    number: '114-2875557-9059409',
+    total: '$17.22',
+    numberItems: 1,
+    orderDate: 'July 4, 2020',
+    deliveryDate: 'July 7, 2020',
+    recipient: 'JS Masher',
+    itemDescription: ['adapter'],
+    itemName: ['Gamecube Controller Adapter'],
+    subtotal: ['$15.98'],
+    buyAgain: true,
+    viewItem: true,
+    productReview: true,
+    productSupport: false
+  },
+  {
+    number: '113-2883177-2648248',
+    total: '$138.93',
+    numberItems: 2,
+    orderDate: 'July 3, 2020',
+    deliveryDate: 'July 5, 2020',
+    recipient: 'JS Masher',
+    itemDescription: [
+      'controller', 'book'
+    ],
+    itemName: [
+      'GameCube Controller', 'The Art of Sql'
+    ],
+    subtotal: [
+      '$94.95', '$33.99'
+    ],
```

Comment:
Rather than storing this data into three separate arrays, you should make one array with an object for each item that stores all data about that item